nixos/tests/pantheon.nix

```diff
@@ -45,8 +45,8 @@ import ./make-test-python.nix ({ pkgs, ...} :
     with subtest("Check if pantheon session components actually start"):
         machine.wait_until_succeeds("pgrep gala")
         machine.wait_for_window("gala")
-        machine.wait_until_succeeds("pgrep wingpanel")
-        machine.wait_for_window("wingpanel")
+        machine.wait_until_succeeds("pgrep io.elementary")
```

Comment:
shouldn't this also be `io.elementary.wingpanel` ?

Reply:
By running `pgrep -l elementary` all I see is `.io.elementary.` and command `pgrep -l wingpanel` outputs nothing on my machine. Shall I change this to something like `ps aux | grep elementary | grep wingpanel`?

Reply:
Thanks! Will fix this.